### ISSUE TITLE
Compatibility with next: 12.3.0

### ISSUE THREAD

**Issue by abusada** [nissy-dev/swc-plugin-react-native-web#23]
Using `"next": "^12.3.0"`

I added the configurations as instructed:
```
 experimental: {
    swcPlugins: [["@nissy-dev/swc-plugin-react-native-web", { commonjs: true }]],
  },
```

but once i run the dev server i start getting the following errors:

![screenshot- 2022-09-27 at 16 46 20@2x](https://user-images.githubusercontent.com/2572320/192559113-9f7d3640-8546-4fc2-bb95-8d62996d1b3c.jpg)

could it be a compatibility issue?

**Comment by abusada** [comment]
that would be highly appreciated

**Comment by nissy-dev** [comment]
Sorry for a late response. I released the new package (0.2.6) for Next.js v12.3.1, so you can use it!
